packages/common/src/utils/testing/testing.tsx

```diff
@@ -54,20 +50,24 @@ export const IntlTestProvider: FC<PropsWithChildren<IntlTestProviderProps>> = ({
   locale,
 }) => {
   useEffect(() => {
+    i18next.changeLanguage(locale);
```

Comment:
reworked slightly - the new `Suspense` is suspending due to `i18next` and the `useEffect` only fires when not suspended. It means that with the original version, i18next would never `init`

This way, it can init on load, and then when the props come down, the language can be set correctly.

Reply:
Just skimmed through React suspend docs, its a bit magically isn't it? i.e. it reads like it interrupts/suspends a non-async method? probably it does some other trickery though, do you have some deeper understanding on how it is implemented?

Reply:
It is a bit tricksy. We were already using it (pre release version was in react v17), so I was surprised to have i18next handle it so differently